### PR TITLE
Search

### PR DESCRIPTION
**Pull request by gcwokeup**

### CODE REVIEW

File: app/assets/stylesheets/search.scss [hunk 1]
```diff
@@ -0,0 +1,14 @@
+.search-results {
+  .avatar {
+    height: 52px;
+    width: 52px;
+    object-fit: cover;
```
Comment (sanghun89) [inline app/assets/stylesheets/search.scss]: just fyi, not even the latest IE supports `object-fit`

File: app/views/search/index.html.erb [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+    <div class="row justify-content-between mt-3">
+      <div class="col title-section">PEOPLE</div>
+      <div class="col body-toned-down text-right">See All</div>
+    </div>
+
+    <div class="row mt-2">
+      <% @users.each do |result| %>
+        <div class="row col-12 col-lg-3">
+          <div class="col-2 text-left">
+            <%= image_tag("home-profile_01.jpg", alt: "rss feed", class: "rounded-circle avatar") %>
```
Comment (sanghun89) [inline app/views/search/index.html.erb]: can we put either a TODO to change this or have duo provide a default avatar User & Organization

File: app/views/search/index.html.erb [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+          </div>
+          <div class="col-10 row">
+            <div class="col-12 heading">
+              <%= result.searchable.name %>
+            </div>
+            <div class="col-12 body-medium-toned-down">
+              Chicago, IL
+              <%#= result.searchable.location %>
```
Comment (sanghun89) [inline app/views/search/index.html.erb]: why is this commented??

File: app/views/search/index.html.erb [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+  <% if @organizations.present? %>
+    <div class="row justify-content-between mt-3">
+      <div class="col title-section">ORGANIZATIONS</div>
+      <div class="col body-toned-down text-right">See All</div>
+    </div>
+
+    <div class="row mt-2">
+      <% @organizations.each do |result| %>
+        <div class="col-2 text-left">
+          <%= image_tag("american_red_cross.jpeg", alt: "rss feed", class: "rounded-circle avatar") %>
```
Comment (sanghun89) [inline app/views/search/index.html.erb]: can we put either a TODO to change this or have duo provide a default avatar User & Organization

File: app/views/search/index.html.erb [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+        </div>
+        <div class="col-10 row">
+          <div class="col-12 heading">
+            <%= result.searchable.name %>
+          </div>
+          <div class="col-12 body-medium-toned-down">
+            Chicago, IL
```
Comment (sanghun89) [inline app/views/search/index.html.erb]: same here

File: app/controllers/search_controller.rb [hunk 6]
```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+class SearchController < ApplicationController
+  def index
+    @search_results = PgSearch.multisearch(params[:query])
+    @users = @search_results.where(searchable_type: "User")
+    @organizations = @search_results.where(searchable_type: "Organization")
```
Comment (sanghun89) [inline app/controllers/search_controller.rb]: also test for this?

File: app/views/search/index.html.erb [hunk 7]
```diff
@@ -0,0 +1,64 @@
...
+  </div>
+
+  <% if @users.present? %>
+    <div class="row justify-content-between mt-3">
+      <div class="col title-section">PEOPLE</div>
+      <div class="col body-toned-down text-right">See All</div>
+    </div>
+
+    <div class="row mt-2">
+      <% @users.each do |result| %>
```
Comment (sanghun89) [inline app/views/search/index.html.erb]: I'd prefer breaking these down and utilize helper methods to section these views in separate components (almost like react), but using smaller helpers and layouts

Comment (sanghun89) [reply app/views/search/index.html.erb]: allows you to test it as well

File: app/views/search/index.html.erb (hunk 2 above)
Comment (gcwokeup) [reply app/views/search/index.html.erb]: gonna add faker profile images so they will all have one

**Comment by markdamobi** [comment]
When I run `rake db:migrate`, there's a change to the schema. The schema change should have already been checked in as part of this pr.

**Comment by markdamobi** [comment]
@gcwokeup would you be able to include a picture of how the search currently works? I'm trying to test the functionality, but it doesn't seem to be working. 
Is this branch still a work in progress? Looks like this pr has been open for a while.

**Pull request closed without merging**